src/test/java/uk/gov/hmcts/darts/dailylist/service/impl/ProcessAllDailyListsForCourthouseTest.java

```diff
@@ -44,11 +45,13 @@ class ProcessAllDailyListsForCourthouseTest {
     private DailyListEntity oldestDailyListEntityForLeeds;
     @Mock
     private DailyListEntity latestDailyListEntityForLeeds;
+    @Mock
+    private LogApi logApi;
 
 
     @BeforeEach
     void setUp() {
-        dailyListProcessor = new DailyListProcessorImpl(dailyListRepository, dailyListUpdater);
+        dailyListProcessor = new DailyListProcessorImpl(dailyListRepository, dailyListUpdater, logApi);
```

Comment:
Can we assert that the logApi method is called appropriately.

Reply:
oh yeah though this was testing something else. The mocking of the processor at the top (plus the name of the class confused me)

Reply:
Done. Changed the class name and eliminated the useless processor mock